prometheus-exporters/ucs-exporter/values.yaml

```diff
@@ -0,0 +1,21 @@
+ucs:
+   enabled: false
+   image:
+      name: ucs-exporter
+      tag: DEFINED-IN-REGION-SECRETS
+   port_number: 9876
+   alerts:
+      enabled: false
+      prometheus: DEFINED-IN-REGION-SECRETS
+   user: DEFINED-IN-REGION-SECRETS
+   master_password: DEFINED-IN-REGION-SECRETS
+   config: config
```

Comment:
still the same problem as before, config does not map to anything here. it needs to be an absolute path in the container like /config/config.yaml

How are we feeding this in? Where is the data coming from?

Reply:
Or is this coming from netbox now?

Reply:
I have added the path to config. its "/ucs-exporter/config". Currently we are creating this config file manually, and I have added the "ucs_hostname_format" var which will get filled at run time with Netbox.